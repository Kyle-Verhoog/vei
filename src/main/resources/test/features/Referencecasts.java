public class Referencecasts {
  public Referencecasts() {}
  public Referencecasts m(Object x) {
    return (A)x;
  }
}