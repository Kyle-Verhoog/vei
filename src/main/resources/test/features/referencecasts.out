PUBLIC ┃ public
CLASS ┃ class
IDENTIFIER ┃ A
{ ┃ {
PUBLIC ┃ public
IDENTIFIER ┃ A
( ┃ (
) ┃ )
{ ┃ {
} ┃ }
PUBLIC ┃ public
IDENTIFIER ┃ A
IDENTIFIER ┃ m
( ┃ (
IDENTIFIER ┃ Object
IDENTIFIER ┃ x
) ┃ )
{ ┃ {
RETURN ┃ return
( ┃ (
IDENTIFIER ┃ A
) ┃ )
IDENTIFIER ┃ x
; ┃ ;
} ┃ }
} ┃ }
